mxd/README.md

```diff
@@ -236,7 +291,80 @@ credential will be able to see it. Technically, that means, that the access poli
 
 > TODO: add link to policy generator tool
 
-> TODO: curl commands
+**Asset:**
+```shell
+curl --location 'http://localhost/alice/management/v3/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "@type": "Asset",
+    "@id": "4", 
+    "properties": {
+        "description": "Product EDC Demo Asset 4"
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "type": "HttpData",
+        "baseUrl": "https://jsonplaceholder.typicode.com/todos/4"
+    }
+}' | jq
+```
+
+**Policy:**
+```shell
+curl --location 'http://localhost/alice/management/v2/policydefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "4",
+  "policy": {
+    "@type": "Policy",
+    "odrl:permission": [
+      {
+        "odrl:action": "USE",
+        "odrl:constraint": {
+          "@type": "LogicalConstraint",
+          "odrl:or": [
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "BusinessPartnerNumber",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "BPNL00000000004X"
+            }
```

Comment:
The first paragraph of this subsection is mentioning that the asset will be only visible for a `dismantler`, but you have added a policy with a BPN constraint. We should keep it consistent. For me it doesn't matter, if `dismantler` or bpn constraint.

Reply:
You're absolutely right. Personally, I find it easier with the BPN at the beginning. Should we rewrite the paragraph or should we limit the policy to a dismantler?